class RobotState:
    ROLE_UNASSIGNED = 1
    ROLE_GOALIE = 2
    ROLE_STRIKER = 3
    ROLE_LEFT_WING = 4
    ROLE_RIGHT_WING = 5
    STATUS_DISCONNECTED = 0
    STATUS_DETERMINING_SIDE = 1
    STATUS_READY = 2
    STATUS_LOCALIZING = 3
    STATUS_WALKING = 4
    STATUS_TERMINATING_WALK = 5
    STATUS_KICKING = 6
    STATUS_FALLEN_FRONT = 7
    STATUS_FALLEN_BACK = 8
    STATUS_FALLEN_SIDE = 9
    STATUS_TRAJECTORY_IN_PROGRESS = 10
    STATUS_PENALIZED = 11
    STATUS_OUT_OF_BOUNDS = 12
    STATUS_STOPPED = 13
